src/fontra/client/web-components/general-settings.js

```diff
@@ -1,4 +1,4 @@
-import { html, css, LitElement } from "https://cdn.jsdelivr.net/npm/lit@2.6.1/+esm";
+import {  html, css, LitElement } from "../third-party/lit.js";
```

Comment:
Prettier has not formatted the double space correctly?

Reply:
I just saw that. Thanks for catching it!
It should work now.